osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -180,18 +180,24 @@ private void addContentToPlaceholder(Drawable content)
 
             if (lastContent != null)
             {
-                lastContent.FadeOut(100, Easing.OutQuint).Expire();
+                lastContent.FadeOut(100, Easing.OutQuint);
```

Comment:
would it not have been simpler and way more understandable to just *not* expire the placeholder? instead of doing whatever that eldritch `BypassAutoSizeAxes` thing is doing?

Reply:
expiring the placeholder and reinstantiating it each time? that would work and save time but would re-retrieve the texture each time, is that fine?

Reply:
```csharp
lastContent.FadeOut(100, Easing.OutQuint);
if (lastContent != notFoundContent)
	lastContent.Expire();
```

Reply:
thought about that, well, since it's local it's fine.